manual_test_nodejs/README.md

```diff
@@ -12,3 +12,10 @@ In the plugin directory:
 `node subprocess.js`
 
 It should stop after 10 seconds with the proper halting message
+
+## Testing resource proxy
+
+`npm start -- --resourceRoutes`
+
+Open <http://localhost:3000/echo/foo/bar?baz=qux> in your browser.  Notice how
+querystrings also work now.
```

Comment:
And please remove this part of the docs :) The CLI option description should suffice. Great initiative updating the docs though !